### PR TITLE
Refactor: Code refactoring according to standards and rules

### PR DESCRIPTION
**Pull request by andrewbess**
### Description (*)

Code refactoring according to standards and rules
I think we should use constants value that correspond standards and rules.
According to Magento db_schema and setup scripts type "VARCHAR" has max length 255.
According to mysql standards, a "TEXT" column with a maximum length of 65,535 (2<sup>16</sup> − 1) characters. The effective maximum length is less if the value contains multibyte characters. Each TEXT value is stored using a 2-byte length prefix that indicates the number of bytes in the value.

### Fixed Issues (if relevant)

1. nope

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by lbajsarowicz — APPROVED** [review]
Great catches! :green_heart: 

Fix Unit Tests and Static tests, by the way, you can improve the names of tests that do not say anything about the fail:
image

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6473](https://jira.corp.magento.com/browse/ENGCOM-6473) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/ImportExport/Model/Import/AbstractEntity.php [hunk 1]
```diff
@@ -15,8 +15,6 @@
 /**
  * Import entity abstract model
  *
- * @api
```
Comment (lbajsarowicz) [inline app/code/Magento/ImportExport/Model/Import/AbstractEntity.php:18]: I'm afraid you cannot just remove `@api`

File: app/code/Magento/ImportExport/Model/Import/Entity/AbstractEntity.php [hunk 2]
```diff
@@ -17,8 +17,6 @@
 /**
  * Import entity abstract model
  *
- * @api
```
Comment (lbajsarowicz) [inline app/code/Magento/ImportExport/Model/Import/Entity/AbstractEntity.php:20]: I'm afraid you cannot just remove `@api`

File: app/code/Magento/CatalogImportExport/Model/Import/Product/Validator.php [hunk 3]
```diff
@@ -71,11 +71,11 @@ protected function textValidation($attrCode, $type)
     {
         $val = $this->string->cleanString($this->_rowData[$attrCode]);
```
Comment (lbajsarowicz) [inline app/code/Magento/CatalogImportExport/Model/Import/Product/Validator.php:72]: How about extracting that to separate method `isTextValid` that will return the result immediately?

There will be no need to use elseif / else, then. The result of isTextValid will be assigned to $valid (I'd rename that to $isValid)

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]

**Comment by andrewbess** [comment]
@lbajsarowicz Perhaps, it was bad idea to refactor this part of core Magento.
How do you think?

**Pull request closed without merging**